packages/contracts/src/Multisig.sol

```diff
@@ -69,15 +74,21 @@ contract Multisig is
 
     /// @notice The [ERC-165](https://eips.ethereum.org/EIPS/eip-165) interface ID of the contract.
     bytes4 internal constant MULTISIG_INTERFACE_ID =
```

Comment:
It is not the old one, but in the new code, the `IProposal`  `createProposal` function was added. 

Note that this `createProposal` function is not needed on the current interface id because it belong to `IProposal`